nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -56,16 +56,17 @@ WaypointFollower::on_configure(const rclcpp_lifecycle::State & /*state*/)
   loop_rate_ = get_parameter("loop_rate").as_int();
```

Comment:
It's worth a migration guide entry to talk about these changes https://github.com/ros-planning/navigation.ros.org/blob/master/migration/Galactic.rst. Please add a new subsection titled `Reduce Nodes and Executors` where you briefly (1-2 sentences) describe the aim of the work and then start a bulleted list of changes made with PR links. You should plan to update that section each PR you submit so that we can keep track in docs the updates and their relevance.